frontend/src/static/css/panels.scss

```diff
@@ -14,6 +14,39 @@
   width: 100%;
   z-index: z-index('app-wrapper');
 
+  #tab-resize {
```

Comment:
Are we using this CSS anywhere else except `resizer.pug`?

Reply:
nope, I placed it there as all the tab related styles are already grouped there though